### ISSUE TITLE
no invalid html bug

### ISSUE THREAD

**Issue by thepassle** [43081j/eslint-plugin-lit#133]
Given the following code:
```html
        <foo-bar
          @current-changed=${this.__offsetChanged}
          page-url="?count=${this.__paginationCount}&offset="
          .count="${Math.ceil(agreements.length / this.__paginationCount)}"
          .current="${this.__paginationOffset}"
        ></foo-bar>
```

I'm getting linting errors on both the lines of `page-url` AND `.count`:

![image](https://user-images.githubusercontent.com/17054057/188816620-d9675436-a7db-4cd7-897a-f2c3ada4dd17.png)

![image](https://user-images.githubusercontent.com/17054057/188816675-7721fcf3-2504-4f15-9f82-36c9be6d7bc8.png)

Changing the value of `page-url` to any string (like for example `page-url="foo"`), passes the lint check. 

Im guessing the template analyzer flags the usage of `&` and `=` in an attribute as invalid, but it seems to also flag the _next_ line as invalid, which seems like a bug.

![image](https://user-images.githubusercontent.com/17054057/188817210-c10c605d-d2ad-4873-a5ca-4358aa293005.png)

**Comment by 43081j** [comment]
for now, you can probably just move the bindings to variables:

```ts
const pageUrl = `?count=${this.__paginationCount}&offset=`;
const count = Math.ceil(agreements.length / this.__paginationCount);

// ...

return html`
  <foo-bar
    page-url=${pageUrl}
    .count=${count}
```

it'll probably be fixed by #130 once i push a new release

**Comment by 43081j** [comment]
Closing since 130 should've fixed this
